lib/rules.json

```
@@ -478,8 +478,8 @@
                                 "&lt;a href=\"https://www.w3.org/2021/Process-20211102/#recs-and-notes\"&gt;Registry track&lt;/a&gt;"
                             ],
                             "customParagraph": true,
-                            "deployment": "It <span class=\"rfc2119\">must</span> include the expections in terms of deployment. The recommended text is: <blockquote>W3C recommends the wide deployment of this registry.</blockquote>",
-                            "stability": "It <span class=\"rfc2119\">must</span> set expectations about the (in)stability of the document. The <span class=\"rfc2119\">recommended</span> text is: <blockquote class=\"boilerplate\"><p>A W3C Registry is a specification that, after extensive consensus-building, has received the endorsement of W3C and its Members.</p>",
+                            "usage": "It <span class=\"rfc2119\">must</span> include the expectations if terms of usage of this registry. The SOTD should include the paragraph: <blockquote>W3C recommends the wide usage of this registry.</blockquote>",
```

there was a typo in my earlier suggestion...

```suggestion
                            "usage": "It <span class=\"rfc2119\">must</span> include the expectations in terms of usage of this registry. The SOTD should include the paragraph: <blockquote>W3C recommends the wide usage of this registry.</blockquote>",
```